tests/testthat/test-ts_plot.R

```diff
@@ -10,66 +10,64 @@ library(testthat)
 
 test_ts_plot <- function() {
   # import helper data
-  data1 <- "https://raw.github.ubc.ca/MDS-2019-20/DSCI_574_spat-temp-mod_students/master/labs/release/lab1/data/ts4_jjshares.csv?token=AAAAOMZQBS5IDU4Q6K3RBXS6NG7NM"
-  data2 <- "https://raw.github.ubc.ca/MDS-2019-20/DSCI_574_spat-temp-mod_students/master/labs/release/lab1/data/ts1_globaltemp.csv?token=AAAAOM4GFOSICJVSVP2YUP26NG7QE"
-  data3 <- "https://raw.github.ubc.ca/MDS-2019-20/DSCI_574_spat-temp-mod_students/master/labs/release/lab1/data/ts6_shorelines.csv?token=AAAAOM2VTEDBLBFBZPV7UW26NHKOU"
-  
-  test_data_1 <- ts_plot(data1, "earnings", 4)
-  test_data_2 <- ts_plot(data2, "temp", 1)
-  
+  time1 <- c("1950 Q1", "1950 Q2", "1950 Q3", "1950 Q4",
+             "1951 Q1", "1951 Q2", "1951 Q3", "1951 Q4",
+             "1952 Q1", "1952 Q2", "1952 Q3", "1952 Q4")
+  earnings1 <- c(0.71, 0.63, 0.82, 0.91,
+                 0.71, 0.63, 0.82, 0.91,
+                 0.71, 0.63, 0.82, 0.91)
+  ts_data1 <- tibble::tibble(time1, earnings1)
+  test_plot1 <- ts_plot(ts_data1, "earnings1", 4)
+
+  time2 <- c("1950", "1951", "1952", "1953",
+             "1954", "1955", "1956", "1957")
+  earnings2 <- c(0.71, 0.63, 0.82, 0.91,
+                 0.51, 1.31, 1.82, 2.01)
+  ts_data2 <- tibble::tibble(time2, earnings2)
+  test_plot2 <- ts_plot(ts_data2, "earnings2", 1)
+
   # tests part
   test_that("The returned plot should be a ggplot object.", {
-    expect_true(ggplot2::is.ggplot(test_data_1))
+    expect_true(ggplot2::is.ggplot(test_plot1))
+    expect_true(ggplot2::is.ggplot(test_plot2))
   })
-  
+
   test_that("The plot for annual time series should contain only 1 line chart", {
-    expect_equal(length(test_data_2$layers), 1)
-    expect_true("GeomLine" %in% c(class(test_data_2$layers[[1]]$geom)))
+    expect_equal(length(test_plot2$layers), 1)
+    expect_true("GeomLine" %in% c(class(test_plot2$layers[[1]]$geom)))
   })
-  
+
   test_that("The plot for quarterly time series should contain 4 subplots", {
-    expect_equal(length(test_data_1$layers), 4)
-    expect_true("GeomLine" %in% c(class(test_data_1$layers[[1]]$geom)))
-    expect_true("GeomSegment" %in% c(class(test_data_1$layers[[2]]$geom)))
-    expect_true("GeomHline" %in% c(class(test_data_1$layers[[3]]$geom)))
-    expect_true("GeomRect" %in% c(class(test_data_1$layers[[4]]$geom)))
+    expect_equal(length(test_plot1$layers), 4)
+    expect_true("GeomLine" %in% c(class(test_plot1$layers[[1]]$geom)))
+    expect_true("GeomSegment" %in% c(class(test_plot1$layers[[2]]$geom)))
+    expect_true("GeomHline" %in% c(class(test_plot1$layers[[3]]$geom)))
+    expect_true("GeomRect" %in% c(class(test_plot1$layers[[4]]$geom)))
   })
-  
+
   test_that("Corresponding error message should be expected if the data path argument doesn't link to a csv file.", {
```

Comment:
this function no takes in a csv. This message is inconsistent with the function.